app/javascript/dashboard/assets/scss/widgets/_conv-header.scss

```diff
@@ -56,6 +56,19 @@ $resolve-button-width: 13.2rem;
       flex-direction: column;
       justify-content: center;
       margin-left: $space-slab;
+      max-width: 20rem;
```

Comment:
Instead of using fixed-width, can we use flex to do the same.  I have created a rough commit for your reference. Please see if this works.

https://github.com/chatwoot/chatwoot/commit/914a32a5255bee9a33e54ca1ea59e0d87889f7c7